app/views/DR/HomeScreen/index.js

```diff
@@ -209,42 +213,60 @@ class HomeScreen extends Component {
                     </View>
                   </View>
                   <View style={styles.actualSituationContainer}>
-                    <Card style={styles.infoCards}>
-                      <Text style={[styles.dataText]}>{confirmed}</Text>
-                      <Text style={styles.text}>
-                        {t('label.positive_label')}
-                      </Text>
-                    </Card>
+                    <TouchableOpacity
+                      style={styles.shadowBorder}
+                      onPress={() => getUrlMap()}>
+                      <Card style={styles.infoCards}>
+                        <Text style={[styles.dataText]}>{confirmed}</Text>
+                        <Text style={styles.text}>
+                          {t('label.positive_label')}
+                        </Text>
+                      </Card>
+                    </TouchableOpacity>
```

Comment:
This piece of code its repeated 4 times in this file and its doing the same thing, just few things change, so you can create a component called Cards or DasboardCard and loop it up passing the parameters they need.
with the creation of the component you can remove getUrlMap function and just add that code to the onPress event.

create an array with the 4 elements or parameters of the card, and pass it to the code.

```
const dashboardCategories = [
      { label: 'label.positive_label', styles: '', value: confirmed },
      {
        label: 'label.deceased_label',
        styles: { color: Colors.BUTTON_LIGHT_TEX },
        value: deaths,
      },
      {
        label: 'label.recovered_label',
        styles: { color: Colors.GREEN },
        value: recovered,
      },
      {
        label: 'label.case_day_label',
        styles: { color: Colors.SUN },
        value: current,
      },
    ];